prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-overview/enrollee-overview.component.html

```diff
@@ -16,11 +16,27 @@
 
   <div class="d-flex justify-content-between align-items-center mt-4">
     <span>
+      <app-alert *ngIf="enrollee?.possiblePaperEnrolmentMatch"
+                 type="warning">
+        <ng-container #alertTitle
+                      class="alert-title">
+          Multiple possible paper enrolment matches found
+        </ng-container>
+        <ng-container #alertContent
+                      class="alert-content">
+          Use the Enrollee search to determine which paper enrolments match
+        </ng-container>
+      </app-alert>
       <div *ngIf="enrolment?.enrollee.gpid"
-            class="mb-4 align-self-center">
+           class="mb-4 align-self-center">
           <strong>GPID:</strong> {{ enrolment?.enrollee.gpid }}
       </div>
-
+      <div *ngIf="enrollee?.linkedEnrolleeId">
+        <a [routerLink]=""
+           (click)="onRoute([enrollee?.linkedEnrolleeId, AdjudicationRoutes.ENROLLEE_OVERVIEW])">
+          {{ (enrolment?.enrollee.gpid | contains: paperEnrolleeGpidFilter) ? 'Linked' : 'Paper' }} Enrolment
+        </a>
+      </div>
```

Comment:
Is the business OK with this difference, as implemented:

![image](https://user-images.githubusercontent.com/7586665/142669643-90c2a1b6-f2d2-4b6f-af1c-988408c10fc9.png)
 
versus mockup:

![image](https://user-images.githubusercontent.com/7586665/142669727-93df1120-f0ae-4eb6-8fce-2d95db7fb596.png)

Reply:
appended the word `link`

Reply:
I mean the lack of the GPID above the hyperlink

Reply:
GPID is only shown when available, ,i.e. after enrolment is approved.